scanner/go.mod

```diff
@@ -134,6 +134,7 @@ require (
 )
 
 replace (
+	github.com/stackrox/rox => ../
 	// The following is a manual copy of replacements done in github.com/stackrox/rox.
```

Comment:
I do wonder how necessary these replacements are

Reply:
Go won't carry them over from `../`, so we must keep them up-to-date to ensure we pull the expected dependencies. Unless you know a way to verify which depends are actually being used by scanner through stackrox?

Reply:
yeah I can't think of anything. Guess we are stuck with this annoyance.

I was originally hoping we would not have to add the rox dependency, but doesn't seem like we can avoid that :(